.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+env:
+  - TOX_ENV=py26
```

Comment:
It might now have. I didn't pay very much attention to what travis told me. I did pay more attention in the following ticket. Where it got fixed.